DigitalLearningSolutions.Web/Controllers/LearningMenuController/LearningMenuController.cs

```diff
@@ -51,9 +51,9 @@ public IActionResult Index(int customisationId)
             if (progressId == null)
             {
                 logger.LogError(
-                    "Redirecting to 500 as no progress id was returned. " +
+                    "Redirecting to 404 as no progress id was returned. " +
                     $"Candidate id: {candidateId}, customisation id: {customisationId}, centre id: {centreId}");
-                return RedirectToAction("StatusCode", "LearningSolutions", new { code = 500 });
+                return RedirectToAction("StatusCode", "LearningSolutions", new { code = 404 });

```

Comment:
Do we not have a test for this which needs updating? If we don't, could you add one? 🙂

Reply:
Yep, added in 149fac0 🙂